l10n_br_account/models/account_invoice_line.py

```diff
@@ -141,27 +141,26 @@ def _compute_price(self):
                 cest=self.cest_id,
                 discount_value=self.discount_value,
```

Comment:
To prevent the core account module to add the discount to the taxes values, couldn't you just to pass zero here?
![image](https://user-images.githubusercontent.com/9100543/111323814-f7e9e400-8648-11eb-985e-b59702c4a287.png)

Reply:
@gabrielcardoso21 acho que percebi este comportamento também. Inclusive acredito que vai o valor do desconto, valor do frete e valor do seguro tbm. Vou testar aqui mas se puder testar ai tbm...